File: Client/tecmo-tourney/src/app/core/models/tournamentStandingModel.ts
import { PrelimTieBreaker } from "src/app/enums";

export interface ITournamentStanding {
    tournamentId: number;
    gamesPlayed: number;
    playerId: number;
    playerName: string;
    preliminariesScore: number;
    preliminaryPosition: number;
    preliminariesTieBreakerUsed?: PrelimTieBreaker | null;
    tournamentFinishPosition: number;
    totalPointsFor: number;
    totalPointsAgainst: number;
    totalPassingYards: number;
    totalRushingYards: number;
    totalPassingYardsAllowed: number;
    totalRushingYardsAllowed: number;
    wins: number;
    loses: number;
  }

File: Service/Models/TournamentStandingModel.cs
namespace TecmoTourney.Models
{
    public class TournamentStandingModel
    {
        public int TournamentId { get; set; }
        public int GamesPlayed { get; set; }
        public int PlayerId { get; set; }
        public string PlayerName { get; set; } = string.Empty;
        public int PreliminariesScore { get; set; }
        public int PreliminaryPosition { get; set; }
        public PrelimTieBreaker? PreliminariesTieBreakerUsed { get; set; }
        public int TournamentFinishPosition { get; set; }
        public int TotalPointsFor { get; set; }
        public int TotalPointsAgainst { get; set; }
        public int TotalPassingYards { get; set; }
        public int TotalRushingYards { get; set; }
        public int TotalPassingYardsAllowed { get; set; }
        public int TotalRushingYardsAllowed { get; set; }
        public int Wins { get; set; }
        public int Loses { get; set; }
        public int Seed { get; set; }
    }
}
